CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -39,7 +40,12 @@ public struct MyCoinView: View {
             footerView
         })
         .task {
-            await viewModel.fetchData(id: myCoinState.id)
+            await viewModel.fetchData(id: myCoinState.id, timeFrameItem: selectedTimeFrameItem)
+        }
+        .onChange(of: selectedTimeFrameItem) { _ in
+            Task {
+                await viewModel.fetchData(id: myCoinState.id, timeFrameItem: selectedTimeFrameItem)
```

Comment:
Wait, I don't think we should call `fetchData` here, because it already reloads the coinDetail information from the server again, but that is not necessary when we change the filter. As we just want to reload the chart only, right? 🤔

Reply:
Good catch 🙌 . I will fix it.

Reply:
I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/99/commits/08a5951398f368ea09c46de2228e482563272bdf 🙏 .